tests/e2e/support/commands.js

```diff
@@ -773,12 +778,19 @@ Cypress.Commands.add('vueRoute', (route) => {
  *   deck?: {suit: number, rank: number}[] deletes all cards except these from the deck
  * }
  */
-Cypress.Commands.add('loadGameFixture', (pNum, fixture) => {
-  return cy
+Cypress.Commands.add('loadGameFixture', (pNum, fixture, gameId = null) => {
```

Comment:
Added `gameId` param for loading fixtures in games you're not watching atm (to join as spectator later)